.github/workflows/ci.yml

```diff
@@ -1,124 +1,159 @@
+
+            - name: Run PHPStan checks
+              run: composer stan
+
+            - name: Check codestyle checks
+              run: composer cs-check
+
+    prefer-lowest:
         runs-on: ubuntu-20.04
         strategy:
             fail-fast: false
             matrix:
-                php: [ '7.4', ]
-                product: [ 'suite' ]
-                composer_preference: [ '', '--prefer-lowest' ]
-        services:
-            rabbitmq:
-                image: rabbitmq
-                ports:
-                    - 5672:5672
-            redis:
-                image: redis
-                ports:
-                    - 6379:6379
-        env:
-            PRODUCT_NAME: ${{ matrix.product }}
-            APPLICATION_ENV: ci.mysql
+                php-version: [
+                        '7.4'
+                ]
+
         steps:
-            -   uses: actions/checkout@v2
-
-            -   uses: getong/mariadb-action@v1.1
-                with:
-                    character set server: 'utf8'
-                    collation server: 'utf8_general_ci'
-                    mariadb version: '10.3'
-                    mysql database: ${{ env.DB_NAME }}
-                    mysql root password: ${{ env.DB_PASSWORD }}
-
-            -   name: Setup PHP
-                uses: shivammathur/setup-php@v2
-                with:
-                    php-version: ${{ matrix.php }}
-                    extensions: mbstring, intl
-                    tools: composer:v2
-
-            -   name: 'Run Elasticsearch'
-                uses: elastic/elastic-github-actions/elasticsearch@master
-                with:
-                    stack-version: 7.6.0
-
-            -   name: Install eco-ci
-                run: git clone https://github.com/spryker-eco/eco-ci.git ecoci
-
-            -   name: Execute test script
-                run: ./ecoci/build/ga.sh ${{ matrix.composer_preference }}
+            - uses: actions/checkout@v2
+
+            - name: Validate composer.json and composer.lock
+              run: composer validate
+
+            - name: Get Composer Cache Directory
+              id: composer-cache
+              run: |
+                  echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - uses: actions/cache@v2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-composer-
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  extensions: mbstring, intl, pdo_mysql
+
+            - name: Composer prefer lowest
+              run: composer lowest-setup
+
+            - name: Run PHPStan (currently not running properly)
+              run: composer stan || true
```

Comment:
do we need the true hack? or does it run here fine?

Reply:
Unfortunately, no. We still need it.